File: Semester_1/RaciocinioAlgoritmico/Tópico 2 - Estruturas de Seleção/soluções/03.py
# Informe o dia da semana a partir de um número entre 1 e 7. Exemplo, 1- Domingo, 2- Segunda etc. Se digitar outro valor deve aparecer “valor inválido”)

numero = int(input("Digite um número entre 1 e 7: "))

if numero == 1:
    print("Domingo")
elif numero == 2:
    print("Segunda")
elif numero == 3:
    print("Terça")
elif numero == 4:
    print("Quarta")
elif numero == 5:
    print("Quinta")
elif numero == 6:
    print("Sexta")
elif numero == 7:
    print("Sábado")
else:
    print("Valor inválido")

File: Semester_1/RaciocinioAlgoritmico/Tópico 2 - Estruturas de Seleção/soluções_extras/03.rs
// Informe o dia da semana a partir de um número entre 1 e 7. Exemplo, 1- Domingo, 2- Segunda etc. Se digitar outro valor deve aparecer “valor inválido”)

use std::io;

fn main() {
    println!("Digite um número entre 1 e 7: ");

    let mut input = String::new();
    io::stdin()
        .read_line(&mut input)
        .expect("Falha ao ler entrada");

    let numero: u32 = match input.trim().parse() {
        Ok(num) => num,
        Err(_) => {
            println!("Valor inválido!");
            return;
        }
    };
    
    match numero {
        1 => println!("Domingo"),
        2 => println!("Segunda"),
        3 => println!("Terça"),
        4 => println!("Quarta"),
        5 => println!("Quinta"),
        6 => println!("Sexta"),
        7 => println!("Sábado"),
        _ => println!("Valor inválido"),
    }
    
}